src/components/tooltip/TooltipDirective.js

```diff
@@ -44,7 +44,8 @@ goog.require('ga_topic_service');
             '<div ng-bind-html="html.snippet"></div>' +
             '<div ng-if="::html.showVectorInfos" class="ga-vector-tools">' +
               '<div ga-measure="::html.feature"></div>' +
-              '<div ga-profile-bt="::html.feature"></div>' +
+              '<div ng-if="::!html.showProfile" ' +
```

Comment:
The display of the button is done by the `ga-profile-bt` directive [see](https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/profile/partials/profile-bt.html).

So I don't think we need this `ng-if` .